app/views/govuk_publishing_components/components/big_number/_value.html.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  number ||= 0
+  show_plus ||= false
+  label ||= nil
+  href ||= nil
+  classes = ["gem-c-big-number__value"]
+  
+  if label.nil? && href
+    classes << "gem-c-big-number__value--decorated"
+  end
+%>
+
+<%= tag.span class: classes do %>
+  <%= number %><%= "+" if show_plus %>
+<% end %>
+
+<% unless label.nil? %>
+  <% # add a virtual space here to handle screen readers printing dictations without a space between the number and the label %>
```

Comment:
This is obviously an important issue - have you considered adding a test for the presence of this element in this situation? Might make it harder to accidentally remove in future.